src/main/java/uk/gov/hmcts/reform/orgrolemapping/controller/WelcomeController.java

```diff
@@ -1,93 +1,24 @@
-import io.swagger.v3.oas.annotations.responses.ApiResponse;
-import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import io.swagger.v3.oas.annotations.Hidden;
 import lombok.NoArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.servlet.view.RedirectView;
-import uk.gov.hmcts.reform.orgrolemapping.domain.model.UserRequest;
-import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.UserType;
-import uk.gov.hmcts.reform.orgrolemapping.domain.service.BulkAssignmentOrchestrator;
-import uk.gov.hmcts.reform.orgrolemapping.v1.V1;
-
-import static org.springdoc.core.Constants.SWAGGER_UI_URL;
-import static uk.gov.hmcts.reform.orgrolemapping.apihelper.Constants.AUTHORIZATION;
-import static uk.gov.hmcts.reform.orgrolemapping.apihelper.Constants.SERVICE_AUTHORIZATION;
 
 @RestController
 @Slf4j
 @NoArgsConstructor
 @Hidden
 public class WelcomeController {
 
-    private BulkAssignmentOrchestrator bulkAssignmentOrchestrator;
-
-
-    @Autowired
-    public WelcomeController(
-            BulkAssignmentOrchestrator bulkAssignmentOrchestrator) {
-        this.bulkAssignmentOrchestrator = bulkAssignmentOrchestrator;
-    }
-
-    @GetMapping(value = "/swagger")
-    public RedirectView swaggerRedirect() {
-        return new RedirectView(SWAGGER_UI_URL, true, false);
-    }
-
+    //This is just a test API
     @GetMapping(value = "/welcome")
     public String welcome() {
         return "Welcome to Organisation Role Mapping Service";
     }
-    //This is just a test API
-
-    @PostMapping(
-            path = "/am/role-mapping/staff/users",
-            produces = V1.MediaType.MAP_ASSIGNMENTS,
-            consumes = {"application/json"}
-    )
-    @ResponseStatus(code = HttpStatus.OK)
-    @Operation(summary = "creates multiple role assignments based upon user profile mapping rules",
-            security =
-            {
-                @SecurityRequirement(name = AUTHORIZATION),
-                @SecurityRequirement(name = SERVICE_AUTHORIZATION)
-            })
-    @ApiResponse(
-            responseCode = "200",
-            description = "OK",
-            content = @Content(schema = @Schema(implementation = Object.class))
-    )
-    @ApiResponse(
-            responseCode = "400",
-            description = V1.Error.INVALID_REQUEST,
-            content = @Content()
-    )
-    public ResponseEntity<Object> createOrgMapping(@RequestBody UserRequest userRequest,
```

Comment:
don't we need this service

Reply:
It has been moved into its own controller